airbyte-integrations/connectors/source-intercom/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: d8313939-3782-41b0-be29-b3ca20d8dd3a
-  dockerImageTag: 0.6.6
+  dockerImageTag: 0.6.7
```

Comment:
@krishan711 new streams are a minor update can you change to 0.7.0 and also bump the version in project.toml and run poetry lock? Thanks